### DISCUSSION TITLE
Option to add Background colour while still persisting pattern

### DISCUSSION THREAD

**Discussion by harshaktg** [bcakmakoglu/vue-flow#1075] [Ideas]
Currently when a bgColor is set for the Background component, it overrides the patters. It'll be great if the pattern can overlay on top of the bgColor.

**Comment by bcakmakoglu** [comment]
You can use simple css to add a bg color instead of using the prop (which will most likely be removed at some point anyway).

Something like this works

```vue
<Background style="background-color: red" pattern-color="#aaa" :gap="8" />
```
